mava/systems/rec_mappo.py

```diff
@@ -593,8 +492,8 @@ def learner_setup(
     init_single = (init_obs_single, init_done)
 
     # Initialise hidden state.
-    init_policy_hstate = ScannedRNN.initialize_carry((config["arch"]["num_envs"]), 128)
-    init_critic_hstate = ScannedRNN.initialize_carry((config["arch"]["num_envs"]), 128)
+    init_policy_hstate = ScannedRNN.initialize_carry((config.arch.num_envs), 128)
+    init_critic_hstate = ScannedRNN.initialize_carry((config.arch.num_envs), 128)
```

Comment:
```suggestion
    nit

```suggestion
    hidden_size = config.system.actor_network.pre_torso_layer_sizes[-1]
    init_policy_hstate = ScannedRNN.initialize_carry((config.arch.num_envs), hidden_size)
    init_critic_hstate = ScannedRNN.initialize_carry((config.arch.num_envs), hidden_size)
```
```